ci/lib/stage-build-gsc.jenkinsfile

```diff
@@ -222,6 +222,32 @@ stage('Build and Test Workloads') {
                 docker rmi verifier:latest gsc-ovms-image &>/dev/null
             '''
         }
+
+
+        try {
```

Comment:
The test is specifically for ubuntu 24.04 but it is currently present under ubuntu 22.04. create a new logic for ubuntu 24.04 and place it under there

Reply:
updated the logic for U24.04